target_decisioning_engine/tests/schema/artifacts/TEST_ARTIFACT_PARAMS.json

```diff
@@ -130,6 +130,43 @@
             ],
             "metrics": []
           }
+        },
```

Comment:
For the most part, we want all the artifacts inside /schema/artifacts to be kept in-sync with [this testing repo](https://github.com/adobe/target-sdk-testing/tree/main/schema/artifacts).  All of the SDKs use the same artifacts so that we can test uniformity across all of them.  Please double check with Jason, but you should probably copy these same changes over to the same files in that repo as well.

Reply:
@dcottingham I have created a PR in that repo for this and requested your review